src/components/challenge-rewards-drawer/ProfileCompletionChecks.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useCallback } from 'react'
+
+import {
+  getUserHandle,
+  getAccountUser
+} from 'audius-client/src/common/store/account/selectors'
+import { getHasFavoritedItem } from 'audius-client/src/components/profile-progress/store/selectors'
+import { profilePage, AUDIO_PAGE } from 'audius-client/src/utils/route'
+import { StyleSheet, View } from 'react-native'
+
+import IconArrow from 'app/assets/images/iconArrow.svg'
+import IconValidationCheck from 'app/assets/images/iconValidationCheck.svg'
+import Text from 'app/components/text'
+import { usePushRouteWeb } from 'app/hooks/usePushRouteWeb'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ThemeColors } from 'app/utils/theme'
+
+import Button, { ButtonType } from '../button'
+
+const messages = {
+  profileCheckNameAndHandle: 'Name & Handle',
+  profileCheckProfilePicture: 'Profile Picture',
+  profileCheckCoverPhoto: 'Cover Photo',
+  profileCheckProfileDescription: 'Profile Description',
+  profileCheckFavorite: 'Favorite Track/Playlist',
+  profileCheckRepost: 'Repost Track/Playlist',
+  profileCheckFollow: 'Follow Five People',
+
+  profileCompletionButton: 'Your Profile'
+}
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    columnContainer: {
+      marginBottom: 32,
+      display: 'flex',
+      flexDirection: 'row',
+      flexWrap: 'wrap'
+    },
+    checkContainer: {
+      display: 'flex',
+      flexDirection: 'row',
+      flexBasis: '50%',
+      marginBottom: 1
+    },
+    checkText: {
+      marginLeft: 8
+    },
+    checkTextDone: {
+      textDecorationLine: 'line-through'
+    },
+    checkCircle: {
+      height: 16,
+      width: 16,
+      borderRadius: 8,
+      borderWidth: 1,
+      borderColor: themeColors.neutralLight4
+    }
+  })
+const renderArrowIcon = color => <IconArrow fill={color} />
+
+export const ProfileCompletionChecks = ({
+  isComplete,
+  onClose
+}: {
+  isComplete: boolean
+  onClose: () => void
+}) => {
+  const handle = useSelectorWeb(getUserHandle)!
```

Comment:
can we avoid the `!`? maybe just a null/no-render check on handle?

Reply:
Updated to just use `currentUser`